python/cherrypy/server.py

```diff
@@ -0,0 +1,19 @@
+import cherrypy
+
+class Root(object):
+    @cherrypy.expose
+    def index(self):
+        return ""
+
+    @cherrypy.expose
+    def user(self, *args):
+        if cherrypy.request.method == "POST" and len(args) == 0:
```

Comment:
This looks weird. What is it that you're trying to achieve?

Reply:
This seem to handle an `POST` request on `/user`

Reply:
@webknjaz I didn't found a way to use rest with method specific requests.
I tried to use popargs (https://docs.cherrypy.org/en/latest/advanced.html#the-popargs-decorator) with MethodDispatcher (https://docs.cherrypy.org/en/latest/tutorials.html#tutorial-7-give-us-a-rest) but didn't work. I had to do this shit if case.

Reply:
I believe it'd be enough to use just a method dispatcher I guess